gulpfile.js

```diff
@@ -0,0 +1,13 @@
+var gulp = require('gulp');
+var replace = require('gulp-replace');
+var pjson = require('./package.json');
+
+gulp.task('propagate-version', function() {
+  gulp.src(['./bower.json'])
+    .pipe(replace(/"version": "\d+\.\d+\.\d+"/, '"version": "' + pjson.version + '"'))
+    .pipe(gulp.dest('./'));
+
+  gulp.src(['./juicy-filedrop.html'])
```

Comment:
Good point... But the Bower one has a fixed "prerelease" name, which also sucks. Any idea how we could get the previous release number and just use it in the regexp?